README.md

```diff
@@ -1,16 +1,17 @@
 <h2>Overview</h2>
 
-Before you deploy a SignalFx lambda wrapper layer to your AWS account, review the available versions and regions for supported languages.
+Before you deploy a Splunk lambda wrapper layer to your AWS account, review the available versions and regions for supported languages.
 
 Additionally, based on your deployment method in AWS, you will need to copy the ARN information. 
 
 ---
 
-<h2>Latest available versions of SignalFx Lambda wrapper layers:</h2>
+<h2>Latest available versions of Splunk Lambda wrapper layers:</h2>
 
 | Supported Languages      | Link to Supported Layer Versions |
 | ----------- | ----------- |
-| Java      | [Click for list of supported layer versions](./java/JAVA.md)     |
+| OTEL Java      | [Click for list of supported layer versions](./otel-java/OTEL-JAVA.md)     |
```

Comment:
I do not see the `otel-java` folder/readme in the repo.

Reply:
@bgola-signalfx looking at the scripts in https://github.com/signalfx/signalfx-lambda-layers I figured out that the first OTEL-JAVA.md along with the containing folder will be added once the layer is deployed. Correct me if I'm wrong, I can easily add "empty" md.

Reply:
I hope @kubawach is right ;) There is a template in signalfx-lambda-layers so if the pipeline passes it should add a new file. Let's run and correct if needed?